driver-sync/src/main/com/mongodb/client/internal/MapReduceIterableImpl.java

```diff
@@ -90,99 +90,100 @@ public void toCollection() {
     }
 
     @Override
-    public MapReduceIterable<TResult> collectionName(final String collectionName) {
+    public com.mongodb.client.MapReduceIterable<TResult> collectionName(final String collectionName) {
         this.collectionName = notNull("collectionName", collectionName);
         this.inline = false;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> finalizeFunction(@Nullable final String finalizeFunction) {
+    public com.mongodb.client.MapReduceIterable<TResult> finalizeFunction(@Nullable final String finalizeFunction) {
         this.finalizeFunction = finalizeFunction;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> scope(@Nullable final Bson scope) {
+    public com.mongodb.client.MapReduceIterable<TResult> scope(@Nullable final Bson scope) {
         this.scope = scope;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> sort(@Nullable final Bson sort) {
+    public com.mongodb.client.MapReduceIterable<TResult> sort(@Nullable final Bson sort) {
         this.sort = sort;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> filter(@Nullable final Bson filter) {
+    public com.mongodb.client.MapReduceIterable<TResult> filter(@Nullable final Bson filter) {
         this.filter = filter;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> limit(final int limit) {
+    public com.mongodb.client.MapReduceIterable<TResult> limit(final int limit) {
         this.limit = limit;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> jsMode(final boolean jsMode) {
+    public com.mongodb.client.MapReduceIterable<TResult> jsMode(final boolean jsMode) {
         this.jsMode = jsMode;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> verbose(final boolean verbose) {
+    public com.mongodb.client.MapReduceIterable<TResult> verbose(final boolean verbose) {
         this.verbose = verbose;
         return this;
     }
 
     @Override
-    public MapReduceIterable<TResult> maxTime(final long maxTime, final TimeUnit timeUnit) {
+    public com.mongodb.client.MapReduceIterable<TResult> maxTime(final long maxTime, final TimeUnit timeUnit) {
         notNull("timeUnit", timeUnit);
         this.maxTimeMS = TimeUnit.MILLISECONDS.convert(maxTime, timeUnit);
         return this;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Do we really need this annotation here and one on the private `action` field above, given that the whole class is annotated with `@SuppressWarnings("deprecation")`? I expect we can remove them.

Reply:
No, we don't.

Done.